app/javascript/src/components/home-title.vue

```diff
@@ -0,0 +1,97 @@
+<template>  
```

Comment:
Trailing spaces not allowed                                                no-trailing-spaces